x-pack/plugins/fleet/server/services/artifacts/artifacts.ts

```diff
@@ -84,55 +88,106 @@ export const createArtifact = async (
   return esSearchHitToArtifact({ _id: id, _source: newArtifactData });
 };
 
-export const bulkCreateArtifacts = async (
-  esClient: ElasticsearchClient,
+// Max length in bytes for artifacts batch
+const MAX_LENGTH_BYTES = 2_000;
```

Comment:
nit: `MAX_LENGTH_BYTES` is a bit confusing in this context. Please rename this to `BULK_CREATE_MAX_ARTIFACTS_BYTES`. If you do rename it I'd suggest also renaming the param names that use this value so they are all named consistently. 

Currently, there is a `maxLengthBytes` and a `createArtifactsBulkBatchSize` that actually hold the same value. Just to avoid forgetting which is which after 3 months. 😅  Perhaps rename all these to `createBulkArtifactsMaxBytes`, or `bulkCreateMaxArtifactsBytes`.

Reply:
done: https://github.com/elastic/kibana/pull/159187/commits/aaf00f7fbe2ee56dc59433931431c74e117b4581